tests/qos/test_qos_sai.py

```diff
@@ -397,6 +414,7 @@ def testQosSaiQSharedWatermark(self, queueProfile, ptfhost, dutTestParams, dutCo
             "pkts_num_leak_out": dutQosConfig["param"][portSpeedCableLength]["pkts_num_leak_out"],
             "pkts_num_fill_min": qosConfig[queueProfile]["pkts_num_fill_min"],
             "pkts_num_trig_drp": triggerDrop,
+            "packet_size": qosConfig[queueProfile]["packet_size"],
```

Comment:
This parameter is not provided for all asics

Reply:
Do you mean for the non-Mellanox ASICs? Should I add them in this PR?

Reply:
yes for non Mellanox asics. use the same logic as in other testcases. update testParams later based on key presence and handle it in the sai_qos_tests.py

Reply:
Fixed.